web-console/src/views/task-view/tasks-view.tsx

```diff
@@ -462,21 +466,27 @@ ORDER BY "rank" DESC, "created_time" DESC`);
             show: supervisorTableColumnSelectionHandler.showColumn('Topic/Stream')
           },
           {
-            Header: 'Status',
+            Header: 'Detailed status',
             id: 'status',
-            accessor: (row) => row.spec.suspended ? 'Suspended' : 'Running',
-            Cell: row => {
-              const value = row.value;
+            width: 300,
+            accessor: (row) => {
+              const { status } = row;
+              if (!status) return '';
+              const { data } = status;
+              if (!data) return '';
+              const { payload } = data;
+              if (!payload) return '';
+              const value = payload.detailedState;
               return <span>
                 <span
-                  style={{ color: value === 'Suspended' ? '#d58512' : '#2167d5' }}
+                  style={{ color: payload.healthy ? '#2167d5' : '#d5100a'}}
```

Comment:
surely there are more statuses (and thus more colors) now. How does healthy relate to detailed status?